### PR TITLE
Preliminary training script

### PR DESCRIPTION
**Pull request by Lunkers**
NB: this isn't done yet, but i want some feedback on the code i have so far

### CODE REVIEW

File: train.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    loss = 0
+    for x,y in testloader:
+        x = x.to(device)
+        z, logdet = model(x)
+        loss += loss_function(x, y)
+    
+#open question: should we add a sample function
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="DD2412 Mini-glow)
```
Comment (Perrrewi) [inline train.py]: missing double quote

**Comment by Lunkers** [comment]
@Dayan-Zhanchi Just a thought, but check if the parameter updates work with the loss function included in this PR
